sql/core/src/main/scala/org/apache/spark/sql/streaming/SourceStatus.scala

```diff
@@ -31,4 +31,4 @@ import org.apache.spark.sql.execution.streaming.{Offset, Source}
 @Experimental
 class SourceStatus private[sql] (
     val description: String,
-    val offset: Option[Offset])
+    val offset: Option[String])
```

Comment:
offset --> offsetDesc, so that it allows us in future to expose actual offsets without the naming being weird.